yt/docs/README.md

```diff
@@ -211,3 +437,47 @@ yt check-permission yql write //tmp
     "subject_name" = "yandex";
 }
 ```
+````
+
+------
+
+### Documentation Philosophy
+
+- When writing text, determine your target audience, whom you are writing for. Take into account that YTsaurus users may have different backgrounds and may be from different areas: analysts, engineers, administrators, and developers. The target audience influences the level of detail of your text and the terminology you use.
+- Your text should not disrupt the integrity of the entire document or its narrative logic. When writing content, consider the context of the section where this content is intended to be added.
+- Try to avoid jargon, slang, Anglicisms, and colloquial language. Use commonly accepted terms. Strive to follow the academic style.
```

Comment:
Lets specify that Anglicisms are for Russian. Also it's a debatable point, I think we are already quite a few and have no desire to stop, since it is closer to the english-sourced naming inside YT (e.g. chunks/tablets/etc.).